### PR TITLE
New feature: WaitForMessageInLogs

### PR DESCRIPTION
**Pull request by MindaugasLaganeckas**
A new feature to wait for a specific message in container logs.

### CODE REVIEW

File: Ductus.FluentDocker/Services/Extensions/ContainerExtensions.cs [hunk 1]
```diff
@@ -280,6 +280,45 @@ public static IContainerService WaitForHealthy(this IContainerService service, l
...
+					if (logs != null && logs.Contains(message))
+						mre.Set();
+					if (stopwatch.ElapsedMilliseconds > millisTimeout)
+					{
+						exception = new FluentDockerException($"Wait for message '{message}' in logs for container {service.Id}");
+						mre.Set();
+					}
+				}, null, 0, 500))
+					
+					mre.Wait();
```
Comment (zionyx) [inline Ductus.FluentDocker/Services/Extensions/ContainerExtensions.cs]:
```suggestion
				mre.Wait();
```

File: Ductus.FluentDocker/Services/Extensions/ContainerExtensions.cs [hunk 2]
```diff
@@ -280,6 +280,45 @@ public static IContainerService WaitForHealthy(this IContainerService service, l
...
+				return null;
+
+			Exception exception = null;
+			var stopwatch = Stopwatch.StartNew();
+			using (var mre = new ManualResetEventSlim())
+			{
+				Timer timer;
+				using (timer = new Timer(_ =>
+				{
+					var logs = service.Logs().Read();
```
Comment (zionyx) [inline Ductus.FluentDocker/Services/Extensions/ContainerExtensions.cs]: All your tabs are belong to us. Great work! :)

Comment (casz) [inline Ductus.FluentDocker/Services/Extensions/ContainerExtensions.cs]: Perhaps an improvement in a separate PR is to allow reading only partial logs using tail `100` as default but allow users to override the default tails length.

Comment (MindaugasLaganeckas) [reply Ductus.FluentDocker/Services/Extensions/ContainerExtensions.cs]: @casz, really nice suggestion! As for the next PR :)

**Comment by mariotoffia** [comment]
Great Work!
It will be part of the next release! I've added Issue #121 to add the _FluentAPI_ part to call this extension when container `State` is set to running.

Cheers,
 Mario :)

**Pull request merged**